checks/raw/sast.go

```diff
@@ -142,13 +141,17 @@ func sastToolInCheckRuns(c *checker.CheckRequest) ([]checker.SASTCommit, error)
 	return sastCommits, nil
 }
 
-func codeQLInCheckDefinitions(c *checker.CheckRequest) ([]checker.SASTWorkflow, error) {
+func getSastUsesWorkflows(
+	c *checker.CheckRequest,
+	useRegex string,
```

Comment:
should this be `usesRegex`? Is this labeled this way because it's applied to the `uses` field? Might be worth a little doc comment.

Reply:
> Is this labeled this way because it's applied to the uses field?

Yes -- I renamed it from `useRegex` to `usesRegex` and added a doc comment.